From ad1cc4044cbe216188a417089f2114df75a705e4 Mon Sep 17 00:00:00 2001
From: Xueling Li <xueling.li@amlogic.com>
Date: Wed, 27 May 2020 16:10:39 +0800
Subject: [PATCH] RDK: migrate code base to trunk [1/1]

PD#SWPL-24384

Problem:
  Westeros-sink inherits GstBaseSink, which will drop key frames with CSD.
  details see SWPL-24203

Solution:
  Integrate parse function inside westeros-sink, and disable it here.

Verify:
  U212 DSTV

Change-Id: I755d539ec947750fbc17ab900b8fe3023d39de97
Signed-off-by: Xueling Li <xueling.li@amlogic.com>
---
 gst-aml-drm-plugins-1.0/configure.ac    | 1 -
 gst-aml-drm-plugins-1.0/src/Makefile.am | 2 +-
 2 files changed, 1 insertion(+), 2 deletions(-)

diff --git a/gst-aml-drm-plugins-1.0/configure.ac b/gst-aml-drm-plugins-1.0/configure.ac
index 3a445a1..9847f61 100644
--- a/gst-aml-drm-plugins-1.0/configure.ac
+++ b/gst-aml-drm-plugins-1.0/configure.ac
@@ -86,7 +86,6 @@ AC_SUBST(GST_PLUGIN_LDFLAGS)
 AC_CONFIG_FILES([Makefile
 src/Makefile
 src/secmem/Makefile
-src/secure_parse/Makefile
 src/dummy/Makefile
 ])
 AC_OUTPUT
diff --git a/gst-aml-drm-plugins-1.0/src/Makefile.am b/gst-aml-drm-plugins-1.0/src/Makefile.am
index bfda5e6..562cf53 100644
--- a/gst-aml-drm-plugins-1.0/src/Makefile.am
+++ b/gst-aml-drm-plugins-1.0/src/Makefile.am
@@ -1 +1 @@
-SUBDIRS = secmem secure_parse dummy
+SUBDIRS = secmem dummy
-- 
2.17.1

